apps/arweave/src/ar_nonce_limiter.erl

```diff
@@ -408,22 +408,27 @@ apply_external_update(Update, Peer) ->
 %%%===================================================================
 
 init([]) ->
-	ok = ar_events:subscribe(node_state),
-	State =
-		case ar_node:is_joined() of
-			true ->
-				Blocks = get_blocks(),
-				handle_initialized(Blocks, #state{});
-			_ ->
-				#state{}
-		end,
-	case ar_config:use_remote_vdf_server() and not ar_config:compute_own_vdf() of
+	case ar_config:verify() of
```

Comment:
I think, it makes more sense to check for auto_join() here

Reply:
Latest commit removes this check and instead uses the `set_dependent_flags` process to disable all vdf features.